pkg/syncer/metadatasyncer.go

```diff
@@ -835,6 +846,405 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	return nil
 }
 
+func initStorageQuotaPeriodicSync(ctx context.Context, metadataSyncer *metadataSyncInformer) {
```

Comment:
Let's error out if we can't initialize; if this has silently failed, it'll be too late to realize something was not working.